TypeCobol.Analysis/Cfg/ControlFlowGraphBuilder.CfgParagraphSymbol.cs

```diff
@@ -0,0 +1,41 @@
+﻿using TypeCobol.Compiler.Scopes;
+using TypeCobol.Compiler.Symbols;
+
+namespace TypeCobol.Analysis.Cfg
+{
+    public partial class ControlFlowGraphBuilder<D>
+    {
+        /// <summary>
+        /// A Paragraph symbol used by a CFG, it contains Sentence symbols.
+        /// </summary>
+        internal class CfgParagraphSymbol : ParagraphSymbol
```

Comment:
Inheriting from `ParagraphSymbol leads to duplictae symbol instanciation as the SemanticDomain currently builds the paragraph symbols too.

Reply:
The inheritance is correct it is a specialization, So you should reproduce were what we suggested : Section and Paragraph symbols can be created by CFG if CFG is enabled or by SDLite otherwise.

Reply:
Solved with PR #1738, CFG does not use Symbols anymore.